.changeset/twenty-deers-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/react": patch
+---
+
+Add components meta data when publish
```

Comment:
@winchesHe as the `scripts` folder is not included in the build we can remove this changeset

Reply:
fixed